clang/lib/Basic/Targets/AVR.cpp

```diff
@@ -336,6 +336,9 @@ static MCUInfo AVRMcus[] = {
     {"attiny1624", "__AVR_ATtiny1624__", "103", 1},
     {"attiny1626", "__AVR_ATtiny1626__", "103", 1},
     {"attiny1627", "__AVR_ATtiny1627__", "103", 1},
+    {"attiny3224", "__AVR_ATtiny1624__", "103", 1},
```

Comment:
It should be a macro `__AVR_ATtiny3234__` for device `attiny3224`, there are similiar issues with `attiny3226` and `attiny3227`.

Reply:
Oops, missed that, fixed...